### PR TITLE
Update User Profile Page

### PR DESCRIPTION
**Pull request by aivantg**
[//]: # "These comments meant for your reference, they are invisible and don't need to be deleted"

## What's new in this PR?

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'

User profile page now has two editing toggle buttons and input validation. styles to come!

[//]: # '############## END ##################'

### How to Review

[//]: # 'The order in which to review files and 
what to expect when testing locally'
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Related PRs

[//]: # "Optional - related PRs you're waiting on
/ PRs that will conflict, etc"
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Next steps

[//]: # "What doesn't work yet, what's NOT in this 
PR/has to be done "
[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
Styling!

[//]: # '############## END ##################'

### Tests Performed, Edge Cases

[//]: # '#####################################'
[//]: # '####### YOUR ANSWER BELOW ###########'
[//]: # '############## END ##################'

### Screenshots

[//]: # '#####################################'
[//]: # '#### YOUR SCREENSHOTS BELOW ########'
image

[//]: # '############## END ##################'

CC: @dfangshuo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # People Power Web App
 
+
```
Comment (dfangshuo) [inline README.md]: nit: uhh unintentional change? lmao

File: src/screens/shared/UserProfile.js [hunk 2]
```diff
@@ -207,133 +238,94 @@ class UserProfile extends React.Component {
...
+                        'updatePermanentState',
+                        contactEditMode
+                      )}
                     </label>
                   </p>
                 </div>
                 <div>
                   <p>
-                    <label htmlFor="updateZip">
+                    <label htmlFor="updatePermanentZipcode">
```
Comment (dfangshuo) [inline src/screens/shared/UserProfile.js]: this is just a label for developers?

File: src/screens/shared/UserProfile.js [hunk 3]
```diff
@@ -66,85 +63,125 @@ class UserProfile extends React.Component {
...
+        ) : (
+          <label className="settings-label">{value}</label>
+        )}
+        {errors[name] && (
+          <label style={{ color: 'red' }}>Error: {errors[name]}</label>
+        )}
+      </div>
+    );
+  }
+
```
Comment (dfangshuo) [inline src/screens/shared/UserProfile.js]: ++ love this modularity!!

File: src/screens/shared/UserProfile.js (hunk 2 above)
Comment (aivantg) [reply src/screens/shared/UserProfile.js]: i'm honestly not sure, i just assumed it might be a css thing and didn't touch it

**Pull request merged**